package-lock.json

```diff
@@ -0,0 +1,2794 @@
+{
+  "name": "invidious-lint",
+  "version": "1.0.0",
+  "lockfileVersion": 3,
+  "requires": true,
+  "packages": {
```

Comment:
Damn, that's a lot of dependencies D:

Reply:
Yeah, there's a lot of transitive dependencies